lib/lev/routine.rb

```diff
@@ -397,7 +402,11 @@ def errors?
     end
 
     def fatal_error(args={})
-      errors.add(true, args)
+      if self.class.raise_fatal_errors?
```

Comment:
I think that makes sense to me. You're using a domain level routine and you tell it you want it to raise fatal errors, that should definitely include nested routines